2_kata.py

```diff
@@ -0,0 +1,8 @@
+#https://www.codewars.com/kata/550554fd08b86f84fe000a58/train/python
+
+def in_array(a1, a2):  
+    #finding all possibles words
```

Comment:
Great job with adding comments, this makes your code more readable and allows others to know what is happening. This is also a great habit to have.